packages/astro/test/params.test.js

```diff
@@ -148,10 +148,4 @@ describe('Astro.params in static mode', () => {
 		const $ = cheerio.load(html);
 		assert.equal($('.category').text(), '%3Fsomething');
 	});
-
-	it("It doesn't encode/decode URI characters such as %25 (%)", async () => {
```

Comment:
This is removed because we can no longer support this. The reason this worked before was we used the result of getStaticPaths during static builds directly. Now we produce paths and send the requests through `app.render(request)`, so the params are in the URL. Those params get decoded for security reasons which we fixed in https://github.com/withastro/astro/security/advisories/GHSA-hr2q-hp5q-x767

These are not encoded in SSR so they no longer can be in static.

Reply:
This requires a `major` changeset, with a thorough explanation of why we can't support this anymore. Worth adding a ticket for that